addons/mediarecorder/userscript.js

```diff
@@ -188,6 +223,67 @@ export default async ({ addon, console, msg }) => {
         })
       );
 
+      // Video resolution.
```

Comment:
I think a better way of doing resolution settings would be to let users pick one of the standard quality measures. (480p, 720p, 1080p, etc.)

If you really wanted, there could also be a "custom" option where you get to choose the exact resolution.

Reply:
For the higher resolutions it won't be the standard width users expect since it's 4:3 and not 16:9.